app/javascript/vue/tasks/people/author_by_letter/components/author_row_component.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <tr v-if="author">
+    <td><a v-html="author.cached" @click="showAuthor"/></td>
+    <td v-if="author.roles[0]"
+        v-html="'list sources for: ' + author.id" @click="showSources"/>
+    <td v-else
+        v-html=" "/>
+    <td v-html="'Uniquify: ' + author.id" @click="uniquify"/>
+    <td>
+      <pin v-if="author.id" :object-id="author.id" :type="author.type"/>
+    </td>
+    <td><span class="button circle-button btn-delete" @click="removeMe()"/></td>
+  </tr>
+</template>
+
+<script>
+
+  import RadialAnnotator from '../../../../components/annotator/annotator'
```

Comment:
@jrflood Can you make these absolute paths, throughout?

Reply:
Done within the scope of this app.